### DISCUSSION TITLE
Support for DEEBOT NEO 2.0 PLUS

### DISCUSSION THREAD

**Discussion by Sam01101101** [MVladislav/bumper#54] [Q&A]
Hi there, thanks for everything you have done so far - its amazing!!

I've replaced a my robot vacuum with DEEBOT NEO 2.0 PLUS which doesn’t seem to be compatible with Bumper - is there plans to support newer devices? 

Thanks,
Sam

**Comment by MVladislav** [comment]
Hi, I don’t have a "DEEBOT NEO 2.0 PLUS" myself, so I’m unable to perform any tests. 
However, any help in improving this project is always welcome! 🙂

I’ve updated the project today to use a newer Python version, which also includes an updated product list for Deebots. 
From what I can see, your Deebot model is now listed in the JSON file, so feel free to give it a try and see if it works for you.

You can check the updated list here:
https://github.com/MVladislav/bumper/blob/dev/bumper/web/plugins/api/pim/productIotMap.json
